src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Allows Peer to Peer transactions
+      key: A
+      order: 0
+    - content: Does not have a centralised authority
+      key: B
+      order: 1
+    - content: Allows for an efficient barter mechanism
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: 922879d6-4e38-4173-880e-94487624c4d0
+  type: SingleChoice
+  content: Each Block of the Blockchain consists of which the following?
+  hint: Blockchain Basics
+  explanation: Each Block contains a hash pointer to the next node , current transaction details , Timestamp.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: A hash Pointer
+      key: A
+      order: 0
+    - content: Transaction Details
+      key: B
+      order: 1
+    - content: Timestamp
+      key: C
+      order: 2
+    - content: All of the above
+      key: D
+      order: 3
+- uuid: 6d73bcc0-02da-4c65-9ff7-9b6571895fe2
+  type: SingleChoice
+  content: Blockchain can be stored as which of the following?
+  hint: NoHint
+  explanation: Blockchain details can be stored as a flat file and a database.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: A flat file
+      key: A
+      order: 0
+    - content: A database
+      key: B
+      order: 1
+    - content: Both of the above
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: f807265d-3208-4a67-8d77-a25bc4bd2596
+  type: SingleChoice
+  content: A block in Blockchain cannot have more than one parent block?
+  hint: NoHint
+  explanation: A block in blockchain cannot have more than one parent because they have unique hash pointer attached to it. Hence , two blocks cannot point to the same block.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1    
+- uuid: 505ebda8-92ba-4922-bd0f-32ebd145b922
+  type: SingleChoice
+  content: Blocks in Blockchain as linked to each to each other .......
+  hint: Linking of the blockchain nodes
+  explanation: The blocks in Blockchain are attached to each other in the backward fashion joining to the previous node and so on.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Backward direction
+      key: A
+      order: 0
+    - content: Forward direction
+      key: B
+      order: 1
+    - content: Not connected
+      key: C
+      order: 2
+- uuid: a6ce806e-02dd-4195-b254-dfb5aa0092f3
+  type: MultipleChoice
+  content: Identify Features of Blockchain?
+  hint: Characteristics of Blockchain
```

Comment:
Dont thin'k `Characteristics of Blockchain` is a relevant Hint. You can add NoHint for this

Reply:
Updated